src/templates/resources-status.html

```
@@ -6,9 +6,9 @@
   'items_status',
   'items_status',
   'Create and modify the status available to resources.'|trans,
-  'Resources status can be associated with a Resource or a Resource Template. They belong to a team and have two main properties: name and color. Examples: In stock, Operational, Maintenance, Processed, etc...'|trans,
+  'Resource status can be associated with a Resource or a Resource Template. They belong to a team and have two main properties: name and color. Examples: In stock, Operational, Maintenance, Processed, etc...'|trans,
   statusArr|filter(c => c.is_current_team == 1),
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)